src/_routes/CommonRoutes.js

```diff
@@ -9,4 +9,5 @@ const rootComponent = isMobile ? TradeMobile : WebCard;
 export default [
     { path: '/', component: rootComponent, onEnter: requireAuthOnEnter },
     { path: 'redirect', component: rootComponent, onEnter: redirectFromOAuth },
+    { path: '*', component: rootComponent, onEnter: requireAuthOnEnter },
```

Comment:
@nuruddeensalihu this will fix the problem, you can copy it at the moment if you need to test in ur machine